api/services/prompt_service.py

```diff
@@ -1,8 +1,9 @@
 # This file is treated as service layer
-from api import models, db
+from api import db
 from datetime import datetime
 from api.helpers import prompt_helper, common_helper, db_helper
 from utils.loggingutils import logger
+from api import models
```

Comment:
We can revert this change if `db` and `models` both are imported from `api`
```suggestion
from api import models, db
from datetime import datetime
from api.helpers import prompt_helper, common_helper, db_helper
from utils.loggingutils import logger
```

Reply:
@Sachinbisht27 I was getting circular import errors due to defining them together, but importing like this solved the issue.